### ISSUE TITLE
CamelCase Css properties get ignored in breakpoints.

### ISSUE THREAD

**Issue by motleydev** [garetmckinley/hedron#330]
This produces no padding.

```jsx
<Grid.Box
  lg={{ width: '50%', paddingRight: '80px' }}
  md={{ width: '100%' }}
>
```

This works fine.
```jsx
<Grid.Box
  lg={{ width: '50%', padding: '80px' }}
  md={{ width: '100%' }}
>
```

**Comment by garetmckinley** [comment]
Thanks for the report! The issue is that currently, the supported styled are [hardcoded here](https://github.com/garetmckinley/hedron/blob/master/src/helpers.js#L43-L59)

I'm going to go ahead and rewrite the `generateStyles()` function to allow more modularity with the styles. I'll update you here :ok_hand:

**Comment by garetmckinley** [comment]
Just released v1.0.1, which has the fix for this in it. Let me know if you have any other issues :smile_cat:
